cfme/tests/containers/test_label.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+
+from cfme.fixtures import pytest_selenium as sel
+from cfme.containers.node import NodeCollection, Node
+from utils import testgen, version
+from cfme.web_ui import CheckboxTable, toolbar as tb
+from utils.appliance.implementations.ui import navigate_to
+from cfme.containers.provider import ContainersProvider
+from utils import conf
+import time
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda provider: version.current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(2)]
+pytest_generate_tests = testgen.generate(
+    [ContainersProvider], scope='function')
+
+# CMP-10572
+
+
+def test_create_label_check(ssh_client, provider):
+    """ This test creates a new label on the master node in Openshift,
+        then it verifies the label exists and is visible on the summary
+        page.There is a sufficient time provided so that CFME is updated
+        with the latest data
+    """
+
+    hostname = conf.cfme_data.get('management_systems', {})[provider.key] \
+        .get('hostname', [])
+    username, password = provider.credentials['token'].principal, \
+        provider.credentials['token'].secret
+
+    ssh_client = ssh_client(
```

Comment:
@gshefer I am finishing up the test that creates labels for all objects and thus will replace the two labels PRs. Indeed, we do need an ssh client fixture for the OCP client, so I'll create that as well. Good call.